interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1720,8 +1720,10 @@ def convert(filename=None, output_name=None, text=None, encoding="latin-1"):
     return len(_species), len(_reactions), surfaces, output_name
 
 
-def main():
-    """Parse command line arguments and pass them to `convert`."""
+def create_parser():
+    """
+    Create argparse parser
+    """
```

Comment:
`create_parser` is a bit ambiguous as a name, given that this entire module is a parser for CTI files. Maybe `create_argparser` would be better?